src/include/attribute.h

```diff
@@ -162,7 +162,7 @@ union attr_val {	      /* the attribute value	*/
 	char 		     *at_str;	/* char string  */
 	struct array_strings *at_arst;	/* array of strings */
 	struct size_value     at_size;	/* size value */
-	pbs_list_head	      at_list;	/* list of resources,  ... */
+	pbs_list_node	      at_list;	/* list of resources,  ... */
```

Comment:
I went through the change and I do not like the renaming of pbs_list_head to pbs_list_node. In current name, i.e., pbs_list_head actually is more readable, and tells me that we are passing a pointer to the head of the linked list, and not some random node in the linked list. Of course, the variable itself is called phead (in most places) signifying that it is indeed the head of the linked list, but having the type itself point that out is actually more readable (ie, if you are looking at prototypes which do not include the parameter name).

In fact the current code defines two types to make things more readable…
typedef struct pbs_list_link {
struct pbs_list_link *ll_prior;
struct pbs_list_link *ll_next;
void	*ll_struct;
} pbs_list_link;
typedef pbs_list_link pbs_list_head;